roles/shiftstack/tasks/deploy_shiftstackclient_pod.yml

```diff
@@ -14,17 +14,31 @@
 # License for the specific language governing permissions and limitations
 # under the License.
 
+- name: Render the pvc manifest
+  ansible.builtin.template:
+    src: templates/shiftstackclient_pvc.yml.j2
+    dest: "{{ (cifmw_shiftstack_basedir, cifmw_shiftstack_client_pod_name + '_pvc.yml') | path_join }}"
+
+- name: Apply the manifest for the PVC creation
+  kubernetes.core.k8s:
+    state: present
+    kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
+    src: "{{ (cifmw_shiftstack_basedir, cifmw_shiftstack_client_pod_name + '_pvc.yml') | path_join }}"
+    wait_condition:
+      type: Ready
+      status: "True"
```

Comment:
I don't think a wait is needed for this PVC. In any case, Anyway, `kubernetes.core.k8s` does nothing for a PVC to wait. Check the doc: https://docs.ansible.com/ansible/latest/collections/kubernetes/core/k8s_module.html#parameter-wait